src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -28,7 +28,8 @@
     static{
         map("-12.0", "Baker Island, Howland Island");
         map("-11.0", "American Samoa, Niue");
-        map("-10.0", "Hawaii, Cook Islands, Marquesas Islands");
+        map("-10.0", "Hawaii, Cook Islands");
+        map("-9.5", "Marquesas Islands");
```

Comment:
How come not all of the time zones in TIME_ZONE_VALUES  have an entry here? For example, -4.5 is missing.